File: e2e/dotnet_projects_net8.0/basic_fsharp/libtest.fs
module BasicFsharp.Tests

open NUnit.Framework
open BasicFsharp

[<TestFixture>]
type LibTests() =
    [<Test>]
    member _.GetMessage_ReturnsExpectedMessage() =
        Assert.AreEqual("Hello from .NET 8.0 F# library!", Lib.getMessage())

File: e2e/dotnet_projects_net8.0/basic_csharp/libtest.cs
using BasicCsharp;
using NUnit.Framework;

[TestFixture]
public sealed class LibTests
{
    [Test]
    public void GetMessage_ReturnsExpectedMessage()
    {
        Assert.AreEqual("Hello from .NET 8.0 library!", Lib.GetMessage());
    }
}
